test/Lib/construct.js

```diff
@@ -225,16 +225,12 @@ var dateList = [
 
     ["toUTCString",0],
 
-    // not in ECMA spec

```

Comment:
Ironically the methods marked "not in ECMA spec" are in the ECMA spec - removed incorrect comment, same in several places below.